src/binfhe/lib/rgsw-acc-lmkcdey.cpp

```diff
@@ -209,7 +208,6 @@ RingGSWEvalKey RingGSWAccumulatorLMKCDEY::KeyGenAuto(const std::shared_ptr<RingG
 
     DiscreteUniformGeneratorImpl<NativeVector> dug;
     NativeInteger Q{params->GetQ()};
-    dug.SetModulus(Q);
```

Comment:
would like to make sure that dug doesn't need to be constructed with the modulus "Q".